.gitignore

```diff
@@ -1,3 +1,6 @@
+data/
```

Comment:
@levicrews why are `data/` and `results/` added to the git ignore? our setup has a root directory within which `code/`, `data/` and `results/` each sit. `data/` and `results/` should not be within the repo so they don't need to be ignored by git. what am i missing?

Reply:
let's discuss. i don't think this pollution would happen in the setup as i see it